resolvers.ts

```diff
@@ -587,8 +587,18 @@ export const resolvers: Resolvers = {
       });
       console.log("entitiesResp - get 1", JSON.stringify(entitiesResp));
 
+      // Non-WGS workflows will not have nextGenSampleId. In this case, return sampleInfo from Rails.
+      const nextGenSampleId = entitiesResp?.data.samples?.[0]?.id;
+      if (!nextGenSampleId) {
+        console.log(`No NextGenSampleId found for railsSampleId: ${args.railsSampleId}`);
```

Comment:
I think we need an eslint-ignore-next-line comment up above this console log?

Reply:
I don't think the fed server has a linter set up (although it probably should)